ui/src/core_components/embed/CorePDF.vue

```diff
@@ -1,17 +1,44 @@
 <template>
-	<div class="CorePDF">
-		<object
-			:key="fields.source.value"
-			class="pdf"
-			:data="fields.source.value"
-			type="application/pdf"
-		>
-			<p>
-				You're not authorised to embed in the browser or you don't have
-				PDF plugin
-				<a :href="fields.source.value">download the PDF file. </a>
-			</p>
-		</object>
+	<div ref="rootEl" class="CorePDF">
+		<div class="controls">
+			<i class="ri-arrow-up-s-line" @click="() => gotoPage(page - 1)"></i>
```

Comment:
For accessibility / semantic HTML purposes I try to use `button` when something is a button. Also, makes them keyboard navigable. Can you please put the `i`s in `button`s?